### PR TITLE
Whitesands Regex/Replacements

### PR DESCRIPTION
**Pull request by MarkSuckerberg**
## About The Pull Request
Replaces all mentions of Waspstation with White Sands for the rebrand. This is probably going to be redone a few times as conflicts arise, so be ready.

## Why It's Good For The Game
Consistency is important

## Changelog
:cl:
code: All mentions of Waspstation have been replaced with White Sands.
/:cl:

### CODE REVIEW

**Comment by WindowsErrors** [comment]
literal whitewashing

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -5,25 +5,21 @@
 # to the same file or dir, add them to the end under Multiple Owners
 
 # BFAT
-/code/modules/power/generator.dm @SynnGraffkin
-/waspstation/code/modules/mapping/minimaps.dm @SynnGraffkin
+/whitesands/code/modules/mapping/minimaps.dm @SynnGraffkin
```
Comment (Maldaris) [inline .github/CODEOWNERS:8]: Now that I think about it, he probably has ownership of spacepods too?

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -10,18 +10,18 @@ If you have difficulty, ask for help in the #coding-general channel on our disco
...
 
 ## Getting Started
 
-WaspStation doesn't have a list of goals and features to add; we instead allow freedom for contributors to suggest and create their ideas for the game. That doesn't mean we aren't determined to squash bugs, which unfortunately pop up a lot due to the deep complexity of the game. Here are some useful starting guides, if you want to contribute or if you want to know what challenges you can tackle with zero knowledge about the game's code structure.
+White Sands doesn't usually have a list of goals and features to add; we instead allow freedom for contributors to suggest and create their ideas for the game. That doesn't mean we aren't determined to squash bugs, which unfortunately pop up a lot due to the deep complexity of the game. Here are some useful starting guides, if you want to contribute or if you want to know what challenges you can tackle with zero knowledge about the game's code structure.
```
Comment (Maldaris) [inline .github/CONTRIBUTING.md:19]: We do actually have a roadmap we could point people to, but idk if that's something we care about too much to put in here.

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -10,18 +10,18 @@ If you have difficulty, ask for help in the #coding-general channel on our disco
...
 If you want to contribute the first thing you'll need to do is [set up Git](https://wiki.boomerstation.space/Setting_up_git) so you can download the source code.
 After setting it up, optionally navigate your git commandline to the project folder and run the command: 'git config blame.ignoreRevsFile .git-blame-ignore-revs'
 
-We have a [list of guides on the wiki](https://wiki.boomerstation.space/index.php/Guides#Development_and_Contribution_Guides) that will help you get started contributing to WaspStation with Git and Dream Maker. For beginners, it is recommended you work on small projects like bugfixes at first. If you need help learning to program in BYOND, check out this [repository of resources](http://www.byond.com/developer/articles/resources).
+We have a [list of guides on the wiki](https://wiki.boomerstation.space/index.php/Guides#Development_and_Contribution_Guides) that will help you get started contributing to White Sands with Git and Dream Maker. For beginners, it is recommended you work on small projects like bugfixes at first. If you need help learning to program in BYOND, check out this [repository of resources](http://www.byond.com/developer/articles/resources).
```
Comment (Maldaris) [inline .github/CONTRIBUTING.md:24]: update the wiki URL plox

**Review by Maldaris — APPROVED** [review]
Couple minor nitpicks, but nothing to prevent merging.

**Review by Superyodeler — APPROVED** [review]

**Pull request merged**